dla_cnn/training_set.py

```diff
@@ -438,31 +438,36 @@ def save_np_dataset(save_file, data):
 # Receives data in the tuple form returned from split_sightline_into_samples:
 # (fluxes_matrix, classification, offsets_array, column_density)
 # Returns indexes of pos & neg samples that are 50% positive and 50% negative and no boarder
-def select_samples_50p_pos_neg(classification):
+from dla_cnn.spectra_utils import get_lam_data
+def select_samples_50p_pos_neg(sightline,kernel):
```

Comment:
Instead of modifying this original method, copy it
into `desi/training_sets.py` and modify it there as needed.

I don't want to make it impossible to go back to the original